gnomad/utils/vcf.py

```diff
@@ -706,11 +706,15 @@ def make_combo_header_text(
     if "subpop" in combo_dict or "pop" in combo_dict:
         if "subpop" in combo_dict:
             header_text = (
-                header_text + f" of {pop_names[combo_dict['subpop']]} ancestry"
+                header_text
+                + f" in the {pop_names[combo_dict['subpop']]} genetic ancestry group"
```

Comment:
```suggestion
                + f" in the {pop_names[combo_dict['subpop']]} genetic ancestry subgroup"
```
I'm not sure I feel strongly about this suggestion, though

Reply:
Yeah I also thought about adding this but wasnt sure on the language, I think this makes sense though